src/olympia/reviewers/models.py

```diff
@@ -13,6 +13,8 @@
 
 import six
 
+from django_extensions.db.fields.json import JSONField
```

Comment:
I keep thinking there's a native `JSONField` now in django but it seems not 🐼

Reply:
There is one... for postgres :disappointed: